lib/guest_installation_and_configuration_base.pm

```diff
@@ -1830,6 +1879,14 @@ sub check_guest_installation_result_via_ssh {
     $self->get_guest_ipaddr if (($self->{guest_ipaddr_static} ne 'true') and (!($self->{guest_ipaddr} =~ /^\d+\.\d+\.\d+\.\d+$/im)));
     save_screenshot;
     if ($self->{guest_ipaddr} =~ /^\d+\.\d+\.\d+\.\d+$/im) {
+        if ($self->{guest_network_type} eq 'virtual_network') {
```

Comment:
Do you think it is possible to reuse $self->{guest_network_device} ? So it can also be set to virtual_network.

Reply:
I am afraid virtual_network should be on upper layer .

Reply:
Actually, for guest in virt-install command line, bridge device and virtual network are equal element. I think you can reuse $self->{guest_network_device}

Reply:
After looking into twice, I think you can keep this way.